conda/recipe/build.sh

```diff
@@ -0,0 +1,19 @@
+#!/bin/bash
+
+env | sort
+
+# TG: The "disabled" features are workaround for possible
+#     conda+configure bugs in library search: building is ok but
+#     linking with the .so doesn't find them (in
+#     conda-forge). Possibly the LD path needs tweaks.
+
+# TODO: re-enable them and see. Also to do: install docs?
+
+./configure --prefix=$PREFIX --enable-shared --disable-python --disable-zlib --disable-external-lapack --disable-external-blas
+make -j4
+make install
+
+cd python
+make pip
+export plumed_default_kernel=$PREFIX/lib/libplumedKernel.so
```

Comment:
I think on MacOS this should be `$PREFIX/lib/libplumedKernel.dylib`

Reply:
Yes and I should test the import...

Reply:
Should be working now. Sorry for the commit spam. To test: `conda install -c tonigi/label/testing plumed2`